src/convertToObject.js

```diff
@@ -1,12 +1,38 @@
 'use strict';
 
+const complexStylesString = require('./stylesString');
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const splited = sourceString.split(';');
+
+  const clear = splited.map((elem) => {
+    return elem.trim();
+  });
```

Comment:
The variable name `clear` is not very descriptive. Consider using a more descriptive name like `trimmedStyles` to improve readability.